skfem/mesh/mesh.py

```diff
@@ -34,6 +34,7 @@ class Mesh:
     # However, some algorithms (e.g., adaptive refinement) require switching
     # off this behaviour and, hence, this flag exists.
     sort_t: bool = False
+    validate: InitVar[bool] = True  # run validation check if log_level<=DEBUG
```

Comment:
How about removing `InitVar` and just using `self.validate` in `__post_init__`?

Reply:
Oh, just let `validate` be a normal property of the object? Sure, that's even simpler!

Reply:
But.... isn't this going to cause excessive validating? I'm pretty sure you want `validate=False` when using `replace`. I don't know all the reasons why to call `replace` on a mesh, but it seems like you'd only want `is_valid` run on whatever the original thing was. What do you want to do?

Reply:
I think it's fine to validate in DEBUG log level just in case. Not all the algorithms we have are perfect.